test/minikube/minikube_tls_admin_test.go

```diff
@@ -153,11 +49,14 @@ func TestKubernetesTLS(t *testing.T) {
 	defer k8s.DeleteNamespace(t, kubectlOptions, namespaceName)
 
 	defer testlib.Teardown(testlib.TEARDOWN_SECRETS)
-	// create the certs...
-	createSecret(t, namespaceName, testlib.CA_CERT_FILE, testlib.CA_CERT_SECRET)
-	createSecret(t, namespaceName, testlib.NUOCMD_FILE, testlib.NUOCMD_SECRET)
-	createSecretWithPassword(t, namespaceName, testlib.KEYSTORE_FILE, testlib.KEYSTORE_SECRET, testlib.SECRET_PASSWORD)
-	createSecretWithPassword(t, namespaceName, testlib.TRUSTSTORE_FILE, testlib.TRUSTSTORE_SECRET, testlib.SECRET_PASSWORD)
+	
+	// create the certs and secrets...
+	tlsCommands := []string{
+		"export DEFAULT_PASSWORD='" + testlib.SECRET_PASSWORD + "'",
```

Comment:
After some more testing, I found that it's indeed needed because otherwise `setup-keys.sh` script won't pick it up.

```
bash-4.2$ DEFAULT_PASSWORD=test
bash-4.2$ setup-keys.sh

: ${CA_STORE:=ca.p12}
: ${SERVER_KEYSTORE:=nuoadmin.p12}
: ${CLIENT_KEYSTORE:=nuocmd.p12}
: ${TRUSTSTORE:=nuoadmin-truststore.p12}

: ${CA_PASSWORD:="$DEFAULT_PASSWORD"}
: ${SERVER_PASSWORD:="$DEFAULT_PASSWORD"}
: ${CLIENT_PASSWORD:="$DEFAULT_PASSWORD"}
: ${TRUSTSTORE_PASSWORD:="$DEFAULT_PASSWORD"}

fail() {
    echo "$1"
    exit 1
}

dname() {
    echo "CN=$1.nuodb.com, OU=Eng, O=NuoDB, L=Boston, ST=MA, C=US"
}

[ -e "$SERVER_KEYSTORE" ] && fail "Server keystore \"$SERVER_KEYSTORE\" already exists"
[ -e "$CLIENT_KEYSTORE" ] && fail "Client keystore \"$CLIENT_KEYSTORE\" already exists"

[ "$CA_PASSWORD" = "" ] && fail "Must define CA_PASSWORD or DEFAULT_PASSWORD"
Must define CA_PASSWORD or DEFAULT_PASSWORD
```